lib/widgets/local_notifications_extension.dart

```diff
@@ -147,7 +147,11 @@ extension LocalNotificationsExtension on MatrixState {
             );
             break;
           case DesktopNotificationActions.openChat:
-            FluffyChatApp.router.go('/rooms/${event.room.id}');
+            if (userId != null) {
+              FluffyChatApp.router.go('/rooms/${event.room.id}?userId=$userId');
```

Comment:
at this stage couldn't we just `setActiveClientWithUserId(userId);` here directly?